connectors/uipath/element-templates/uipath-prototype-get-item-status.json

```diff
@@ -0,0 +1,294 @@
+  },
+  "documentationRef": "https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/rest/",
+  "category": {
+    "id": "connectors",
+    "name": "Connectors"
+  },
+  "appliesTo": [
+    "bpmn:Task"
+  ],
+  "elementType": {
+    "value": "bpmn:ServiceTask"
+  },
+  "groups": [
+    {
+      "id": "authentication",
+      "label": "Authentication"
+    },
+    {
+      "id": "endpoint",
+      "label": "Endpoint Get Item Status"
+    },
+    {
+      "id": "timeout",
+      "label": "Connect Timeout"
+    },
+    {
+      "id": "output",
+      "label": "Response Mapping"
+    },
+    {
+      "id": "errors",
+      "label": "Error Handling"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:http-json:1",
+      "binding": {
+        "type": "zeebe:taskDefinition:type"
+      }
+    },
+    {
+      "label": "Type",
+      "id": "authenticationType",
+      "group": "authentication",
+      "description": "Choose the authentication type",
+      "value": "oauth-client-credentials-flow",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Bearer Token Auth",
+          "value": "bearer"
+        },
+        {
+          "name": "OAuth 2.0",
+          "value": "oauth-client-credentials-flow"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.type"
+      }
+    },
+    {
+      "id": "method",
+      "label": "Method",
+      "group": "endpoint",
+      "type": "Hidden",
+      "value": "get",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "method"
+      }
+    },
+    {
+      "label": "URL",
+      "description": "Request URL to get the status of the task. Please, replace '{{cloudUrl}}', '{{cloudOrg}}' and '{{cloudTenant}}' with the the values from your uipath orchestrator.",
+      "group": "endpoint",
+      "type": "String",
+      "feel": "required",
+      "value": "=\"https://{{cloudUrl}}/{{cloudOrg}}/{{cloudTenant}}/odata/Jobs",
+      "editable": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "url"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "pattern": {
+          "value": "^(=|https?://).*",
+          "message": "Must be a http(s) URL."
+        }
+      }
+    },
+    {
+      "label": "Filters",
+      "description": "Map of query parameters to add to the request URL",
+      "group": "endpoint",
+      "type": "String",
+      "feel": "required",
+      "value": "{\"$Filter\":\"State eq 'Successful' AND Id eq \"+itemId}",
```

Comment:
why we are filtering for successful here? I think just getting the status is fine: "{"$Filter":"Id eq "+itemId}"

Reply:
Done! Thank you!